morf-excel/src/test/java/org/alfasoftware/morf/excel/TestTableOutputter.java

```diff
@@ -79,7 +79,7 @@ public void testTableOutputWithFieldsPopulated() throws WriteException {
     // Given
     Record record1 = mock(Record.class);
     when(record1.getString("Col1")).thenReturn("STRING VALUE");
-    when(record1.getBigDecimal("Col2")).thenReturn(new BigDecimal(12.34));
+    when(record1.getBigDecimal("Col2")).thenReturn(BigDecimal.valueOf(12.34));
```

Comment:
Note: `new BigDecimal(double)` differs from `BigDecimal.valueOf())`, this is a potential change!

This is a test, so we are fine.